tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -119,8 +120,8 @@ def test_timeout_on_pre_010_brokers(self, broker_version):
 
         processor.node.account.ssh(processor.start_cmd(processor.node))
         with processor.node.account.monitor_log(processor.STDERR_FILE) as monitor:
-            monitor.wait_until('Exception in thread "main" org.apache.kafka.streams.errors.BrokerNotFoundException: Could not find any available broker.',
-                               timeout_sec=60,
-                               err_msg="Never saw 'no available brokers' error message " + str(processor.node.account))
+            monitor.wait_until('Could not read message within 45 seconds, maybe could not a find broker',
```

Comment:
does this change make sense, as we disable the test anyway?

Reply:
Actually, that's when I was trying something else.  I overlooked removing it.